sklearn/cluster/_agglomerative.py

```diff
@@ -755,6 +755,8 @@ class AgglomerativeClustering(ClusterMixin, BaseEstimator):
     n_connected_components_ : int
         The estimated number of connected components in the graph.
 
+        .. versionchanged:: 0.21
```

Comment:
versionchanged really needs some description of how the behaviour has changed. And I think it should be used only rarely, when it would really help the user to understand a substantial change  across versions.